Vue InstantSearch/server-side-rendering/src/entry-server.js

```diff
@@ -23,8 +23,7 @@ export default context => {
           context.algoliaState = instantsearch.getState();
 
           resolve(app);
```

Comment:
would maybe make more sense if it was removed here, to stay with the Vue CLI example

Reply:
sure thing.